### ISSUE TITLE
Hunner master on libterm ios 

### ISSUE THREAD

**Issue by aymanioo** [aymanioo/Libterm#1]
Hunner master lets me add the email and when I try and use the password list included, I get this error. Python3: command not found and then it selects the password list, however it kicks me back out of the hunner master

Enter password list:
Hunner»Mail»Password»
python3: command not found
Password list: password/password_list.txt
Ayman’s iPad $

**Comment by adrianhoh** [comment]
Me also same also same like that

**Comment by aymanioo** [comment]
Yeah it doesn't appear to be working even if I changed the path to the directory and chmod +x
